README.md

```diff
@@ -4,24 +4,29 @@ Tools for managing RealDeviceMap data
 
 ## rdm-tools.php
 ### Changes
-* Now you can report on all nests at once!
-* You don't have to have spawns/stops visible on map to perform reporting anymore.
-* Revised optimization function - same super simple algorithm, might be better performing though
-* Changes to data storage, functions no longer loop through map objects, coordinates for gyms/stops/spawns are stored separately. Should help with performance
+* spawnpoint color is different for verified despawn time
```

Comment:
can you fix this section's grammar/punctuation to match the rest of the document?